drivers/i2c/Kconfig.i2c_emul

```diff
@@ -5,6 +5,7 @@ config I2C_EMUL
 	bool "I2C emulator"
 	default y
 	depends on DT_HAS_ZEPHYR_I2C_EMUL_CONTROLLER_ENABLED
+	select EMUL
```

Comment:
Instead of selecting `EMUL` I could also change the `EMUL` config to be default y if one of these emulated bus controllers are enabled but I prefer the `select` here.